app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/data/CbmChatRepository.kt

```diff
@@ -351,8 +362,17 @@ internal class CbmChatRepositoryImpl(
   }
 
   private suspend fun Raise<String>.uploadMediaToBotService(uri: Uri): String {
+    val fileSize = contentResolver.openFileDescriptor(uri, "r")?.use {
+      it.statSize
+    } ?: raise("Could not determine file size for uri: $uri")
```

Comment:
Do we want to exit out eagerly if we can't figure out the size, even though it might've been just fine to upload the file anyway if it was not that big?
Do we ever expect `statSize` to fail here for some reason, or are we just being careful just in case?